Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -4527,13 +4527,10 @@ void MainWindow::slotMeHearMyself(bool checked/*=false*/)
 
 void MainWindow::slotMeEnableVoiceActivation(bool checked)
 {
-    ttSettings->setValue(SETTINGS_GENERAL_VOICEACTIVATED, checked);
-    enableVoiceActivation(checked, SOUNDEVENT_VOICEACTMEON, SOUNDEVENT_VOICEACTMEOFF);
-    if (QObject::sender() == ui.actionEnableVoiceActivation)
-        addTextToSpeechMessage(TTS_TOGGLE_VOICETRANSMISSION, (checked?tr("Voice activation enabled"):tr("Voice activation disabled")));
+    enableVoiceActivation(checked, SOUNDEVENT_VOICEACTMEON, SOUNDEVENT_VOICEACTMEOFF, QObject::sender() == ui.actionEnableVoiceActivation?true:false);
 }
 
-void MainWindow::enableVoiceActivation(bool checked, SoundEvent on, SoundEvent off)
+void MainWindow::enableVoiceActivation(bool checked, SoundEvent on, SoundEvent off, bool TTS)

```

Comment:
What is the purpose of boolean value "TTS"? I don't see it being used anywhere.

Reply:
Can't you change the code so only ui.actionEnableVoiceActivation calls MainWindow::slotMeEnableVoiceActivation() ?